db/migrate/20200128183708_add_must_update_fields.rb

```diff
@@ -0,0 +1,8 @@
+class AddMustUpdateFields < ActiveRecord::Migration[6.0]
+  def change
+    create_table :settings do |t|
+      t.string :key
+      t.string :value
```

Comment:
I would add a non-empty validation (and their tests), what do you think?

Reply:
Also, I would add an index by key :).

Reply:
I don't agree on the index, given amount of records it will make things slower due to generating and maintaining index

Reply:
I don't agree. The index will be generated only during the insert and I don't think configuration keys will be added frequently but they should be asked often.

Reply:
We are talking about performance and indexing in a 2-3 rows table. If you still think is an improvement feel free to benchmark and see there will be no significant benefit if it even is